tests/test_exporters.py

```diff
@@ -0,0 +1,112 @@
+import os
+import unittest
+from io import StringIO
+import filecmp
+import tempfile
+import csv
+from django.core.exceptions import ValidationError
+from django.core.management import call_command
+from django.test import TestCase
+
+from linguatec_lexicon.models import (DiatopicVariation, Entry, Example,
+                                      GramaticalCategory, Lexicon, Region,
+                                      VerbalConjugation, Word)
+
+class ExporterDataTestCase(TestCase):
+    LEXICON_NAME = 'es-ar'
+    LEXICON_CODE = 'es-ar'
+
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        Lexicon.objects.create(
+            name=cls.LEXICON_NAME, src_language='es', dst_language='ar',
+        )
+
+    def setUp(self):
+        # importdata requires that GramaticalCategories are initialized
+        base_path = os.path.dirname(os.path.abspath(__file__))
+        sample_path = os.path.join(base_path, 'fixtures/gramcat-es-ar.csv')
+        call_command('importgramcat', sample_path, verbosity=0)
+
+    def test_export_data(self):
+        
+
+        base_path = os.path.dirname(os.path.abspath(__file__))
+        sample_path = os.path.join(base_path, 'fixtures/sample-input.xlsx')
+        call_command('importdata', sample_path, self.LEXICON_NAME)
+
+
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            call_command('exportdata', self.LEXICON_CODE, tmpdirname + '/test-output-data-file.csv')
+            sample_path = os.path.join(base_path, 'fixtures/export_test_files/export_data_expected_result.csv')
+
+            result=[]
+            with open(sample_path) as resultfile:
+                reader = csv.reader(resultfile, delimiter=';')
+                for i, row in enumerate(reader):
+                    result.insert(i,row)
+            with open(tmpdirname + '/test-output-data-file.csv') as outfile:
+                reader = csv.reader(outfile, delimiter=';')
+                for i, row in enumerate(reader):
+    
+                    self.assertEqual(result[i],row)
```

Comment:
To compare two files, we could test this: 
```
self.assertListEqual(
    list(io.open(sample_path)),
    list(io.open(tmpdirname + '/test-output-data-file.csv')))
```
See: https://stackoverflow.com/questions/42512016/how-to-compare-two-files-as-part-of-unittest-while-getting-useful-output-in-cas

Reply:
Thank you for this suggestion, definitely it is so useful, so much better than the previous code